webservices/legal_docs/current_murs.py

```diff
@@ -8,6 +8,8 @@
 from webservices.rest import db
 from webservices.utils import create_eregs_link, get_elasticsearch_connection
 from webservices.tasks.utils import get_bucket
+from webservices.legal_docs.utils import get_subjects, get_election_cycles, get_participants, get_sorted_respondents, get_disposition, get_documents, get_open_and_close_dates
```

Comment:
1. It would be better to use syntax like https://github.com/18F/openFEC/blob/develop/webservices/legal_docs/__init__.py#L6-L16 to avoid long lines. 
2. Sorting the individual imports would make them easier to manage.